external/date.js

```diff
@@ -228,7 +228,7 @@ $.extend( _Date.prototype, {
 	equal: function( other ) {
```

Comment:
Also, is this method still used anywhere? I don't see any calls in `ui/widgets/calendar.js`.